mage_ai/server/execution_manager.py

```diff
@@ -0,0 +1,72 @@
+from distutils.dir_util import copy_tree
+from mage_ai.data_preparation.models.pipeline import Pipeline
+from typing import Callable
+import multiprocessing
+import os
+import shutil
+
+'''
```

Comment:
Let's put the comments below the class name and use double quotes to wrap the comments.